src/binutil/server.rs

```diff
@@ -434,8 +434,12 @@ fn pre_start(cfg: &TiKvConfig) {
 }
 
 fn check_system_config(config: &TiKvConfig) {
+    let mut rocksdb_max_open_files = config.rocksdb.max_open_files;
+    if config.rocksdb.titan.enabled {
+        rocksdb_max_open_files *= 2;
```

Comment:
please add a comment here

Reply:
Is it possible that Titan has more files than RocksDB @Connor1996

Reply:
Done. 

> please add a comment here

Reply:
> Is it possible that Titan has more files than RocksDB @Connor1996

Looks like Titan uses the same max open files setup as rocksdb does. 

![image](https://user-images.githubusercontent.com/3982329/60692268-c201dc00-9f07-11e9-98be-f714f75a8c22.png)